spring-boot-service/variables.tf

```diff
@@ -0,0 +1,58 @@
+variable "name" {
+  type = string
+}
+
+variable "port" {
+  type    = number
+  default = 8080
+}
+
+variable "autoscaling" {
+  type = object({
+    min_number_of_instances = optional(number, 1)
+    max_number_of_instances = optional(number, 1)
```

Comment:
Hmm, jeg veit ikke om det er feil, men hvis vi setter dette, tror du ikke det vil føre til at mange kjører i prod med 1 instans? Som vi ikke anbefaler på en spring-boot-service

Reply:
Kanskje ha 3 som default da? Tenkte mest på at man ikke skal ende opp med å kjøre med 3 i test. Men her kunne vi jo også satt default basert på miljø (altså 3 som default i prod of 1 som default i test)? Eller blir det for mye logikk inne i modulen?